app/components/SideBar/AccountsList/AccountsList.jsx

```diff
@@ -0,0 +1,45 @@
+import { FormattedMessage as T } from "react-intl";
+import { Balance } from "shared";
+import style from "./AccountsList.module.css";
+import { classNames } from "pi-ui";
+
+const isImported = (accountNumber) => accountNumber === Math.pow(2, 31) - 1;
+
+const AccountsList = ({ isShowingAccounts, balances }) => (
+  <div
+    className={style.sidebarMenuTotalBalanceExtended}
+    style={{ display: isShowingAccounts ? "flex" : "none" }}>
```

Comment:
Would be nice to avoid inline styling, I suggest using conditioned classes

Reply:
Done.